src/node_crypto.cc

```diff
@@ -3684,8 +3684,8 @@ void Hmac::HmacInit(const FunctionCallbackInfo<Value>& args) {
 bool Hmac::HmacUpdate(const char* data, int len) {
   if (!initialised_)
     return false;
-  HMAC_Update(&ctx_, reinterpret_cast<const unsigned char*>(data), len);
-  return true;
+  int r = HMAC_Update(&ctx_, reinterpret_cast<const unsigned char*>(data), len);
+  return r == 1;
```

Comment:
And https://github.com/openssl/openssl/commit/2dc769a1c17e1e0c7ae

I think this routine can return error codes, not just `1`, so the check should be against `!= 0`.